src/logic/application-code/dotenv/dotenv.ts

```diff
@@ -27,22 +28,25 @@ export function createNewDotenv({
   parameters,
   security,
   token,
+  logEnabled,
 }: {
   previousDotenv?: string;
   providerName: string;
   parameters?: IntegrationParameter[];
   security?: SecurityScheme[];
   token?: string | null;
+  logEnabled?: boolean;
 }): NewDotenv {
   const previousContent = previousDotenv ?? '';
 
   const parameterEnvs = getParameterEnvs(providerName, parameters);
   const securityEnvs = getSecurityEnvs(providerName, security);
   const tokenEnv = makeTokenEnv(token);
+  const logEnv = makeLogEnv(logEnabled === true ? '"on"' : '"off"');
```

Comment:
The motivation to change the default value is that user could have it already set up (e.g. using `export` in the terminal)?

Reply:
I also don't see the point for this condition right now if the goal was to promote the env :)